_benchmark/user-guide/contributing-workloads.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: default
+title: Contributing custom workloads
+nav_order: 11
+parent: User guide
+---
+
+# Contributing custom workloads
+
+After [creating a custom workload]({{site.url}}{{site.baseurl}}/benchmark/creating-custom-workloads/), you can contribute your workload for other OpenSearch Benchmark users to use. If you plan on contributing a workload, use the following guidelines.
+
+## README and license
+
+Each workload must include a detailed `README.MD` file, which should include the following:  
+
+- The purpose of the workload and which users should use the workload. When creating a description, consider the specific use case of the workload and how the workload's use case is different from other workloads included in the [workloads repository](https://github.com/opensearch-project/opensearch-benchmark-workloads/).
+- Make sure that any data included in the workload's dataset does not contain any proprietary data or Personal Identifiable Information (PII). 
```

Comment:
How is this relevant for the readme file?

Reply:
Because of the example document that must be included in the README.

Reply:
Then maybe you can first list the output sample bullet and then add this as a note under it? Otherwise, it feels like a requirement on the content in the workload and not on the README itself. Or make it a note after the first paragraph that discusses sharing the workload?